External/Plugins/ASCompletion/Model/PathModel.cs

```diff
@@ -584,11 +587,6 @@ public void RemoveFile(string fileName)
             }
         }
 
-        public void Touch()
-        {
-            LastAccess = DateTime.Now;
```

Comment:
Yes the idea initially was to use the `LastAccess` info to keep recent cps in memory.